lib/resque/scheduler/cli.rb

```diff
@@ -68,7 +68,8 @@ class Cli
         },
         {
           args: ['-q', '--quiet', 'Run with minimal output [QUIET]'],
-          callback: ->(options) { ->(q) { options[:quiet] = q } }
+          # CLIO: Improved true/false handling, which was causing us to not log stuff.
```

Comment:
I'm not sure I understand this, I played around with optparse a little bit and it doesn't appear that this even takes an argument, so shouldn't `q` always be true or false?

Reply:
TBH, I only vaguely recall this, but it's something we've added in Manage: https://github.com/clio/themis/blob/master/config/resque_scheduler_startup.rb#L12-L17

I figured I'd port all the monkey-patches over while I was at it. Maybe it was a bug that went away in later versions of resque-scheduler? I'll check.

Reply:
Aha! This is why we did that: https://github.com/resque/resque-scheduler/issues/565

Looks like it's been fixed in the current version and I can remove this code.

Reply:
Except... it's fixed in the current version of _Resque_, not resque-scheduler, so we'd have to upgrade to a version of Resque that's more recent than November 2021 to get the fix.